makefiles/tinydtls_config/dtls_config.h

```diff
@@ -0,0 +1,10 @@
+// Not that we'd need *that* code, but if neither ECC nro PSK is defined,
+// there's empty unions and such (and newer verisons of the library complain
+// more explicitly)
```

Comment:
Is there a reason for the C++ style comments here? Would you mind changing this to C-style comments?

Reply:
Reason? Yes[1]. Good one? No.

[1]: I learned C++ before learning C, and they always worked so those are what I do. I first used C style comments in a project that used Doxygen, so unless a commit hook or reviewer complains, I write C++ style for inline actual comments, and C-style for documentation.

Anyway, the latest fixups do away with them.